parts/linux/cloud-init/artifacts/labels.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+# Update Labels for this Kubernetes node
```

Comment:
nit: better to be more precise if possible.

Reply:
```Updates Labels for this Kubernetes node (adds any missing, and updates and to newest values)```